giving-wheel/app/components/surprise.tsx

```diff
@@ -0,0 +1,197 @@
+import { Transition, Dialog } from "@headlessui/react";
+import React, {Fragment, useEffect, useState} from "react";
+import {Form, useFetcher} from "@remix-run/react";
+import Loading from "~/components/loading";
+
+// todo: add success messages
```

Comment:
Don't leave todo comments, it's better to just remove them

Reply:
we had some things we weren't able to do for our project so instead of leaving them undone just as-is, we spun it so that they were left as "exercises" for the reader to do (lol 😆) and talked about it in the `README`.
https://github.com/daffyorg/examples/tree/main/Good%20Neighbor#-inspiration-

i think leaving the TODOs would help in that case for them to quickly jump in, but if you include a bit more detail as to what they need to do in the `README` as a guide. otherwise I agree, it'd be better to just remove them